config/GregTech/GregTech.cfg

```diff
@@ -484,6 +609,52 @@ undergroundfluid {
 
     }
 
+    miranda {
+        # Dimension ID or Class Name
+        S:Dimension=Miranda
+
+        hydrogen_sulfide {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=100
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=5
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount=600
```

Comment:
this could be increased, since it needs processing, maybe 800-1000

Reply:
change to 900